graphql_doc/sample_queries.md

```diff
@@ -0,0 +1,114 @@
+# **GraphQL**
+
+## **Sample Queries & Mutations**
+
+Here are few sample queries to speed up your developing/testing process.
+*Note: thanks for your patience as the platform is always evolving, there might outdated sample queries. Please let us know if you find a mistake and/or outdated information.*
+
+1. profile
+```
+query {
+		profile(
+      input:
+      { account: "someuser" })
+  {
+	account
+   avatar
+   bio
+   chatToken
+   chatUserId
+   createdAt
+   createdBlock
+   createdEosAccount
+   email
+   interests
+   invitations
+   location
+   name
+  }
+}
+```
```

Comment:
A couple of notes on queries code snippets.

1. The snippets are not properly formatted. It's pretty simple to fix, just go to `https://staging.cambiatus.io/api/graphiql` and press the `Prettify` button.
2. GitHub support syntax highlighting for GraphQl queries, just use <code>```graphql</code> in the code snippets.

It may look like this:

```graphql
{
  profile(input: {account: "lucca"}) {
    account
    avatar
    bio
    chatToken
    chatUserId
    createdAt
    createdBlock
    createdEosAccount
    email
    interests
    # invitations # this field causes an error
    location
    name
  }
}
```

And the actual markdown is:

<pre>
```graphql
{
  profile(input: {account: "lucca"}) {
    account
    avatar
    bio
    chatToken
    chatUserId
    createdAt
    createdBlock
    createdEosAccount
    email
    interests
    # invitations # this field causes an error
    location
    name
  }
}
```
</pre>

Reply:
+1